src/backend/distributed/planner/multi_router_planner.c

```diff
@@ -1412,68 +1410,19 @@ TargetEntryChangesValue(TargetEntry *targetEntry, Var *column, FromExpr *joinTre
  * shard-extended deparsed SQL to be run during execution.
  */
 static Job *
-RouterInsertJob(Query *originalQuery, Query *query, DeferredErrorMessage **planningError)
+RouterInsertJob(Query *originalQuery, Query *query)
```

Comment:
The function comment seems obsolete: `query against the provided shard interval` 
which we don't provide